mu/locale/ru_RU/LC_MESSAGES/mu.po

```diff
@@ -0,0 +1,1618 @@
+msgid "Show the music used by Pygame Zero."
+msgstr "Показать музыку, используемую Pygame Zero."
+
+#: mu/modes/pygamezero.py:124
+msgid "Stop your Pygame Zero game."
+msgstr "Остановите игру Pygame Zero."
+
+#: mu/modes/python3.py:133
+msgid "Python 3"
+msgstr "Python 3"
+
+#: mu/modes/python3.py:135
+msgid "Create code using standard Python 3."
+msgstr "Пишите код, используя стандартный Python 3."
+
+#: mu/modes/python3.py:151 mu/modes/python3.py:200
+msgid "Run your Python script."
+msgstr "Запустите свой скрипт Python."
+
+#: mu/modes/python3.py:157
+msgid "Debug"
+msgstr "Отладить"
+
+#: mu/modes/python3.py:158
+msgid "Debug your Python script."
+msgstr "Отладьте свой скрипт Python."
+
+#: mu/modes/python3.py:165
+msgid "Use the REPL for live coding."
+msgstr "Используйте REPL для живого программирования."
+
+#: mu/modes/python3.py:175
+msgid "Plot data from your script or the REPL."
+msgstr "Постройте данные из вашего скрипта или REPL."
+
+#: mu/modes/python3.py:208
+msgid "Stop your Python script."
+msgstr "Остановить свой скрипт Python."
+
+#: mu/modes/python3.py:280
+msgid "Starting iPython REPL."
+msgstr "Запуск iPython REPL."
+
+#: mu/modes/python3.py:285
+msgid "Stopping iPython REPL (this may take a short amount of time)."
+msgstr "Остановка iPython REPL (это может занять некоторое время)."
+
+#: mu/modes/python3.py:377
+msgid "REPL started."
+msgstr "REPL запущен."
+
+#: mu/modes/python3.py:386
+msgid "REPL stopped."
+msgstr "REPL остановлен."
+
+#: mu/modes/web.py:58
+msgid "Web"
+msgstr "Веб"
+
+#: mu/modes/web.py:60
+msgid "Build simple websites with the \"Flask\" web framework."
+msgstr "Создавайте простые веб-сайты с помощью веб-фреймворка Flask."
+
+#: mu/modes/web.py:76 mu/modes/web.py:129
+msgid "Run the web server."
+msgstr "Запустите веб-сервер."
+
+#: mu/modes/web.py:83
+msgid "Open your website in a browser."
+msgstr "Откройте свой сайт в браузере."
+
+#: mu/modes/web.py:89
+msgid "Templates"
+msgstr "Шаблоны"
+
+#: mu/modes/web.py:90
+msgid "Load HTML templates used by your website."
+msgstr "Загрузите HTML-шаблоны, используемые вашим сайтом."
+
+#: mu/modes/web.py:96
+msgid "CSS"
+msgstr "CSS"
+
+#: mu/modes/web.py:97
+msgid "Load CSS files used by your website."
+msgstr "Загрузите файлы CSS, используемые вашим сайтом."
+
+#: mu/modes/web.py:104
+msgid "Open the directory containing images used by your website."
+msgstr "Откройте каталог, содержащий изображения, используемые вашим сайтом."
+
+#: mu/modes/web.py:137
+msgid "Stop the web server."
+msgstr "Остановить веб-сервер."
+
+#: mu/modes/web.py:157
+msgid "This is not a Python file!"
+msgstr "Это не файл Python!"
+
+#: mu/modes/web.py:158
+msgid ""
+"Mu is only able to serve a Python file. Please make sure the current tab in Mu "
+"is the one for your web application and then try again."
+msgstr ""
+"Mu может обслуживать только файл Python. Убедитесь, что текущая вкладка в Mu - "
+"это вкладка вашего веб-приложения, а затем повторите попытку."
```

Comment:
Incorrect translation, I think this will work:
```suggestion
"Mu может запускать только файлы Python. Убедитесь, что текущая вкладка в Mu - "
"это вкладка вашего веб-приложения, а затем повторите попытку."
```
Hard to find a correct translation for "serve" given the context this word is used in.